### PR TITLE
Pool size per image

### PR DESCRIPTION
**Pull request by Gentux**
Add possibility to define poolSize per image and not globally.

If image's pool size is not define, it take the default one

### CODE REVIEW

**Comment by dynamiccast** [comment]
Documentation needs to be updated as well.

**Comment by Gentux** [comment]
Which part of documentation ? Are you talking about http://documentation.nanocloud.com/docs

Those docs are not here.

**Comment by dynamiccast** [comment]
Sorry for the confusion. I was referring to the API documentation. We need to add the `poolSize` field in the Image model.

File: api/services/MachineService.js [hunk 1]
```diff
@@ -505,12 +503,16 @@ function updateMachinesPool() {
             let machineCreated = _.find(machinesCount.rows, (m) => m.image === image.id) || {count: 0};
             let machineToCreate = 0;
             let machineToDestroy = 0;
+            let machinePoolSize = config.machinePoolSize;
+            if (image.poolSize === null) {
+              machinePoolSize = config.machinePoolSize;
```
Comment (corentindrouet) [inline api/services/MachineService.js:508]: This if is useless, But i think the first assignation should be: machinePoolSize = image.poolSize ?

**Review by Leblantoine — APPROVED** [review]
This PR works.

**Review by corentindrouet — APPROVED** [review]

**Pull request merged**